interp/interp_test.go

```diff
@@ -1055,6 +1084,9 @@ func TestInterp(t *testing.T) {
 
 		// Then test it in GoAWK
 		t.Run(testName, func(t *testing.T) {
+			if f, ok := flags["windows"]; !f && ok && runtime.GOOS == "windows" {
+				t.Skipf("skipping windows")
+			}
```

Comment:
Per comment above, we'll need to include this logic in the `runAWK` section above too.